client/src/modules/list/components/Items/ListItem/ListItem.jsx

```diff
@@ -59,10 +65,10 @@ class ListItem extends PureComponent {
     if (!isMember) {
       return;
     }
-
+    const formattedName = formatName(editedBy, formatMessage);
```

Comment:
`editedBy` is current user name - do we have to call `formatName` on the current User name? current user can not be anonymous...